tmpfiles.sh

```diff
@@ -70,8 +88,16 @@ relabel() {
 	status=0
 	for path in ${paths}; do
 		if [ -e "${path}" ]; then
+			if ! owned_by_root $1 ; then
+				echo "cowardly refusing to relabel" >&2
+				return 108
+			fi
 			if [ -x /sbin/restorecon ]; then
 				dryrun_or_real restorecon ${CHOPTS} "${path}" || status="$?"
+				if [ $status -ne 0 ]; then
```

Comment:
use braces with `${status}`

Reply:
Changed in commit 4f4576a6781fb64a6c798d131f8309802c2f9c3a